src/app/hiv-care-lib/surge-report/surge-report-base.component.ts

```diff
@@ -0,0 +1,167 @@
+import { Location } from '@angular/common';
+import * as _ from 'lodash';
+import * as Moment from 'moment';
+import * as rison from 'rison-node';
+
+import { Component, OnInit, Output, Input } from '@angular/core';
+import { ActivatedRoute, Router, Routes } from '@angular/router';
+import { SurgeResourceService } from 'src/app/etl-api/surge-resource.service';
+
+@Component({
+  selector: 'surge-report-base',
+  templateUrl: './surge-report-base.component.html',
+  styleUrls: ['./surge-report-base.component.css'],
+})
+export class SurgeReportBaseComponent implements OnInit {
+  public params: any;
+  public indicators: string;
+  public selectedIndicators = [];
+  public surgeWeeklyReportSummaryData: any = [];
+  public columnDefs: any = [];
+  public enabledControls = 'weekControl';
+  public reportName = 'Surge Weekly Report';
+  public currentView = 'weekly';
+  public isReleased = false;
```

Comment:
We don't need this variable, as it was used for MOH report where we have unreleased and released reports

Reply:
used this variable to hide the daily surge report controls